commit bb61ea9ed2bcecd0dbc63af9ee1c07b30c5b387c
Author: Michael Still <mikal@stillhq.com>
Date:   Mon Sep 14 18:32:13 2026 +1000

    Convert the core services to JSON logging.
    
    Glance and Placement now honour openstack_logging_json, in the same
    one line form as Keystone. Glance has two configuration files, the API
    and the cache, and both need it or the cache logs would be the one
    thing in the deployment the parser could not read.
    
    No per service variable is added, for the reason given when Keystone
    was converted: every oslo.log service is tailed by a single Fluentd
    input with a single parser, so the format cannot vary between them.
    
    Change-Id: Iadbb097c884fe4f6659719828681161bc2b73e1c
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/ansible/roles/glance/templates/glance-api.conf.j2 b/ansible/roles/glance/templates/glance-api.conf.j2
index aa6a3e8f5..bda046aad 100644
--- a/ansible/roles/glance/templates/glance-api.conf.j2
+++ b/ansible/roles/glance/templates/glance-api.conf.j2
@@ -1,6 +1,7 @@
 {# j2lint: disable=operator-enclosed-by-spaces #}
 [DEFAULT]
 debug = {{ glance_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 # NOTE(elemoine) log_dir alone does not work for Glance
 log_file = /var/log/kolla/glance/glance-api.log
diff --git a/ansible/roles/glance/templates/glance-cache.conf.j2 b/ansible/roles/glance/templates/glance-cache.conf.j2
index 4fc6400ca..99af4610d 100644
--- a/ansible/roles/glance/templates/glance-cache.conf.j2
+++ b/ansible/roles/glance/templates/glance-cache.conf.j2
@@ -1,6 +1,7 @@
 [DEFAULT]
 
 debug = {{ glance_logging_debug }}
+use_json = {{ openstack_logging_json }}
 log_file = /var/log/kolla/glance/glance-cache.log
 
 image_cache_max_size = {{ glance_cache_max_size }}
diff --git a/ansible/roles/placement/templates/placement.conf.j2 b/ansible/roles/placement/templates/placement.conf.j2
index 552862fdf..d1d34df65 100644
--- a/ansible/roles/placement/templates/placement.conf.j2
+++ b/ansible/roles/placement/templates/placement.conf.j2
@@ -1,5 +1,6 @@
 [DEFAULT]
 debug = {{ placement_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_file = /var/log/kolla/placement/{{ service_name }}.log
 
